.tav.yml

```diff
@@ -210,16 +210,19 @@ hapi-no-async-await:
   name: hapi
   versions: '>=9.0.1 <17.0.0'
   commands:
-    - node test/instrumentation/modules/hapi/basic.js
+    - node test/instrumentation/modules/hapi/basic-legacy-path.js
     - node test/instrumentation/modules/hapi/set-framework.js
 hapi-async-await:
   name: hapi
   node: '>=8.2'
   versions: '>=17.0.0'
   commands:
-    - node test/instrumentation/modules/hapi/basic.js
+    - node test/instrumentation/modules/hapi/basic-legacy-path.js
     - node test/instrumentation/modules/hapi/set-framework.js
-
+'@hapi/hapi':
+  node: '>=8.2'
+  versions: '>=17.0.0'
+  commands: node test/instrumentation/modules/hapi/basic.js
```

Comment:
You need to add `set-framework-2.js` to this list of commands as well (just as above)